### ISSUE TITLE
Power consumption not accurate

### ISSUE THREAD

**Issue by M1dgard** [jaagr/polybar#1250]
[`powertop`](https://en.wikipedia.org/wiki/PowerTOP) and [`powerstat`](http://kernel.ubuntu.com/~cking/powerstat/) show a consumption of around 12 Watts. Polybar's battery module shows around 150 Watts.

After looking at the code, this seemed like it might be useful information:
There's no `/sys/class/power_supply/BAT0/current_now`, there is a `power_now`.
To give an idea of the deviation, here are some numbers. I varied my power usage and watched the Polybar indicator (configured to update once a second, but it actually updated once in ~5s). When it changed, I noted its value and the value Powerstat (configured to report once a second) reported. This is not really accurate because of the delays due to the manual labour, and the fact that the readings were not fully synchronised.
```
PB  powerstat PB-PS  PB/PS
140   11.97    128   11.7
140   11.95    128   11.7
138   11.95    126   11.5
159   13.97    145   11.4
163   16.21    147   10.1
207   17.66    189   11.7
207   17.11    190   12.1
197   16.68    180   11.8
176   15.10    161   11.7
522   45.4     477   11.5
555   49.4     506   11.2
454   40.8     413   11.1

          avg  216   11.5
          std  138   0.5

```

Polybar version: 3.1.0-git

**Comment by NBonaparte** [comment]
What about the output of `/sys/class/power_supply/BAT0/power_now`? To convert to watts, the raw value needs to be divided by 10^6 IIRC.

**Comment by patrick96** [comment]
This looks like #928, which was fixed in #958, which is not part of a release yet. You'll have to wait for 3.2.0 or compile polybar against the newest commit. Can you confirm that you get the right values compiling against the newest commit?

**Comment by M1dgard** [comment]
Yes, it's fixed! Awesome. Thanks.
